packages/components/ng-at-internet/package.json

```diff
@@ -42,4 +42,4 @@
   "peerDependencies": {
     "angular": "~1.6"
   }
-}
\ No newline at end of file
+}
```

Comment:
Prettier ? Files should not be impacted in this PR though

Reply:
It's since I am rebased on `develop` that these new lines appear